src/NzbDrone.Core/Datastore/Migration/076_add_flac_cue.cs

```diff
@@ -0,0 +1,15 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(076)]
+    public class add_flac_cue : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Tracks").AddColumn("IsSingleFileRelease").AsBoolean().WithDefaultValue(false);
```

Comment:
The tracks table is mainly just metadata, do we need this column added?

Reply:
No this should be a WIP leftover, I'll remove it.